pkg/io/jsonio/unmarshal.go

```diff
@@ -350,20 +356,14 @@ func unmarshal(filename string) (*spec.Spec, error) {
 	return jsonSpec, err
 }
 
-func parseOverlappingVpcs(cidr ipblock.IPBlock, vpcs map[ir.ID]*ir.VPCDetails) ([]ir.ID, error) {
+func parseOverlappingVpcs(cidr ipblock.IPBlock, vpcs map[ir.ID]*ir.VPCDetails) []ir.ID {
```

Comment:
```suggestion
func parseOverlappingVpcs(cidr *ipblock.IPBlock, vpcs map[ir.ID]*ir.VPCDetails) []ir.ID {
```
Better stick to the convention of passing pointers to IPBlocks